### DISCUSSION TITLE
How I configured TCE and deployed my first real workload in my homelab

### DISCUSSION THREAD

**Discussion by vimwtf** [vmware-tanzu/community-edition#2838] [Show and tell]
Hi all,

I'm pretty new to TCE, and my only real experience thus far with Kubernetes in general has been guided workshops and training labs. I feel like I don't really get a great understanding of a new-to-me solution until I've actually used it to address a need. With that in mind, I spent some time over the past week getting TCE set up on my NUC homelab and then deployed phpIPAM to it. I really enjoyed the learning experience and I look forward to diving deeper in the coming weeks.

I took notes along the way and shared those on my blog in case my experience can help anyone else in getting started with TCE:
https://www.virtuallypotato.com/tanzu-community-edition-k8s-homelab/

Cheers,
John

**Comment by stmcginnis** [comment]
Awesome write up @jbowdre!

**Comment by jpmcb** [comment]
This is awesome!!! 🚀

**Comment by qnetter** [comment]
Great stuff, John! Mind if we link to it in a future update of tanzucommunityedition.io?

**Reply by vimwtf** [reply]
@qnetter please feel free - I'd be honored!
